hedera-node/src/main/java/com/hedera/services/txns/crypto/helpers/AllowanceHelpers.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * -
+ *  * ‌
+ *  * Hedera Services Node
```

Comment:
Multiple files have an extra level of asterisks in the license headers.

Reply:
Thanks @steven-sheehy . It is my IntelliJ formatting. Fixed it.